src/GroupFactory.php

```diff
@@ -16,16 +16,14 @@ public function __construct(ContainerInterface $container = null)
 
     public function __invoke(?string $prefix = null, array $routes = []): Group
     {
-        $group = new Group($prefix, function (RouteCollectorInterface $collector) use ($routes) {
+        $group = new Group($prefix, function (Group $group) use ($routes) {
```

Comment:
Either the change is not correct or an interface should be deleted (there's only a single implementation anyway).

Reply:
you're right

Reply:
> Either the change is not correct or an interface should be deleted (there's only a single implementation anyway).

https://github.com/yiisoft/router/blob/master/src/Group.php#L23